src/main/java/org/sagebionetworks/bridge/validators/ValidatorUtils.java

```diff
@@ -239,4 +245,24 @@ private static final long convertPeriod(Period period, Function<Duration, Long>
         }
     }
 
+    public static final void validateStringLength(Errors errors, int maxLength, String persistingText, String fieldName) {
+        if (persistingText == null || persistingText.isEmpty()) {
+            return;
+        }
+        if (persistingText.length() > maxLength) {
+            errors.rejectValue(fieldName, String.format(INVALID_STRING_LENGTH, maxLength));
+        }
+    }
+    
+    public static <T> void validateJsonLength(Errors errors, int maxLength, T persistingObject, String fieldName) {
+        if (persistingObject == null) {
+            return;
+        }
+        try {
+            String jsonString = BridgeObjectMapper.get().writeValueAsString(persistingObject);
+            validateStringLength(errors, maxLength, jsonString, fieldName);
+        } catch (JsonProcessingException ignored) {
+            // TODO: Decide what to do with an error converting the JSON
```

Comment:
This should be impossible since we just deserialized it to store it as JsonNode in the first place, so I would throw a BridgeServerException (500 error) as something is very wrong.

Reply:
Added the bridge exception in the catch block.

Reply:
Looks good